packages/cli/src/config/PublishDetails.ts

```diff
@@ -189,7 +189,7 @@ const validateLocalizableResources = (config: PublishDetails) => {
 const checkIconDimensions = async (iconPath: string): Promise<boolean> => {
   const size = await runImgSize(iconPath);
 
-  return size?.width != size?.height || (size?.width ?? 0) > 512;
+  return size?.width != size?.height || (size?.width ?? 0) != 512;
```

Comment:
We don't currently have any guidance for this on https://docs.solanamobile.com/dapp-publishing. Can you work with @Michaelsulistio to publish something with an effective date, after which we can include this change?

Reply:
Removed this code. Will followup to release guidance first.